.github/workflows/kubeflow.yml

```diff
@@ -24,20 +27,29 @@ jobs:
       # Workaround from FuseML (https://github.com/fuseml/fuseml/blob/main/.github/workflows/ci.yml)
       # as the TF images are too large for the GH action runner disk
       - name: Free disk space
+        shell: bash
         run: |
           echo "Available storage:"
           df -h
           echo
           echo -n "  Removing: .NET (frees ~22GB)"
           sudo rm -rf /usr/share/dotnet
           echo "... done"
-          echo -n "  Removing: Android"
-          sudo rm -rf /usr/local/lib/android
-          echo "... done"
           echo
           echo "Available storage:"
           df -h
 
+      - name: Install Linux System Dependencies
+        if: runner.os=='Linux'
+        run: sudo apt install graphviz
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
```

Comment:
Just out of curiosity, how can this work without using any secrets to authenticate? Or are those secret keys hardcoded somewhere and just not visible in this yaml file?